pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -63,11 +75,26 @@ def on_train_batch_start(
 
         device = trainer.strategy.root_device
         device_stats = trainer.accelerator.get_device_stats(device)
+
+        if self.cpu_stats and device.type != "cpu":
+            # Don't query CPU stats twice if CPU is accelerator
+            device_stats.update(get_cpu_process_metrics())
```

Comment:
can you add somethings info about it in the docs?
if you are not sure where, let me know. I'll push a commit for that :)